File: Dihya/scripts/cli_onboarding.py
# CLI Python – Onboarding utilisateur
import requests, argparse
parser = argparse.ArgumentParser()
parser.add_argument('--email', required=True)
parser.add_argument('--lang', default='fr')
args = parser.parse_args()
res = requests.post('http://localhost:8001/api/onboarding/', json={'email': args.email, 'lang': args.lang})
print(res.json())

File: Dihya/scripts/cli_onboarding.js
// CLI Node.js – Onboarding utilisateur
const axios = require('axios');
const argv = require('yargs').option('email', { demandOption: true, type: 'string' }).option('lang', { default: 'fr', type: 'string' }).argv;
axios.post('http://localhost:8002/api/onboarding/', { email: argv.email, lang: argv.lang })
  .then(res => console.log(res.data))
  .catch(err => console.error(err.response?.data || err.message));
